awscrt/http.py

```diff
@@ -337,17 +343,74 @@ def request(self,
                 on_response: Optional[Callable[..., None]] = None,
                 on_body: Optional[Callable[..., None]] = None,
                 manual_write: bool = False) -> 'Http2ClientStream':
+        """Create `Http2ClientStream` to carry out the request/response exchange.
+
+        NOTE: The HTTP stream sends no data until `Http2ClientStream.activate()`
```

Comment:
What is the envisioned lifecycle for a stream? Are we activating and closing every transaction or do we send many requests?